### PR TITLE
Emali fixes, styles and much more.

### PR DESCRIPTION
**Pull request by stargazer39**

### CODE REVIEW

File: src/api/register/index.js [hunk 1]
```diff
@@ -143,3 +158,28 @@ const generateFileName = () => {
 		".jpg"
 	);
 };
+
+const dataURItoBlob = (dataURI) => {
+	// convert base64 to raw binary data held in a string
+	// doesn't handle URLEncoded DataURIs - see SO answer #6850276 for code that does this
+	var byteString = atob(dataURI.split(',')[1]);
```
Comment (rusiruavb) [inline src/api/register/index.js:165]: Please use `let` keyword to define variables

File: src/views/AwarenessSession/index.jsx [hunk 2]
```diff
@@ -39,20 +39,26 @@ function AwarenessSession() {
...
                     await sendEmail(member_data.email, "Mini hackathon awareness session", str);
 
-                    setStatus({ state: "success", message: "Success, You have successfully registered for the Awareness Session."});
+                    setStatus({ state: "success", message: "Success, You have successfully registered for the Awareness Session." });
 
                     setTicket(prevTicket => { return { ...prevTicket, display: true }});
 
                     reset();
                 }catch(error){
+                    console.error(error);
```
Comment (rusiruavb) [inline src/views/AwarenessSession/index.jsx:61]: Remove the `console.log`

File: src/views/Register/MemberForm2.jsx [hunk 3]
```diff
@@ -30,10 +31,12 @@ function MemberForm2({ formKey, handleSubmitFunc }) {
                 })();
             })
         });
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (rusiruavb) [inline src/views/Register/MemberForm2.jsx:34]: Remove these eslint comments

File: src/views/Register/NameForm.jsx [hunk 4]
```diff
@@ -6,28 +6,31 @@ import { useEffect } from "react";
...
             return new Promise((resolve, reject) => {
                 handleSubmit((data) => {
-                    resolve(data.teamName);
+                    resolve(data);
                 }, () => {
                     resolve(null);
                 })();
             })
         });
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (rusiruavb) [inline src/views/Register/NameForm.jsx:28]: Remove eslint comments

File: src/views/Register/index.jsx [hunk 5]
```diff
@@ -6,10 +6,12 @@ import NameForm from "./NameForm";
...
 	const [status, setStatus] = useState({ state: "none", message: "" });
 	const [currentIndex, setCurrentIndex] = useState(0);
-	const form_count = 5;
+
+	// Members are scalable using the default value. (5 forms, with the first one being team name and team count.)
+	const [memberCount, setMemberCount] = useState(4);
 
 	const [submitFunctions, setSubmitFunctions] = useState({});
-	const [teamInfo, setTeamInfo] = useState({});
+	const [, setTeamInfo] = useState({});
```
Comment (rusiruavb) [inline src/views/Register/index.jsx:14]: If `setTeamInfo` is not using, please remove it from the code.

**Review by rusiruavb — CHANGES_REQUESTED** [review]

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]
Remove all the console logs

File: src/views/Register/index.jsx (hunk 5 above)
Comment (stargazer39) [reply src/views/Register/index.jsx:14]: Acctually to manage the state, I need it, if you note the setTeamInfo function in line 43, I'm using it to get the previous state in the code, but I'm not using the acctual teamInfo variable in the code.

File: src/views/Register/NameForm.jsx (hunk 4 above)
Comment (stargazer39) [reply src/views/Register/NameForm.jsx:28]: There's a bit issue here. I need this bit of code to run only once in the function, but not use all the vairables in the dependency array, it I do that, it leads to a infinity loop. I wonder why I can't just pass the empty array [] and not see the lint error

**Comment by stargazer39** [comment]
Ohh I'll remove them

**Comment by stargazer39** [comment]
I did the changes

File: src/views/Register/NameForm.jsx (hunk 4 above)
Comment (stargazer39) [reply src/views/Register/NameForm.jsx]: I removed them anyway

**Review by senuravihanjayadeva — APPROVED** [review]

**Pull request merged**